test/document-cases/html-block-elements.ecmarkdown

```diff
@@ -0,0 +1,21 @@
+<div>This is a div *no emd*</div>
+*this is emd*
+*so is this*
+
+<section>*no emd here*
+  *but emd is allowed here*
+</section>*but not here*
```

Comment:
In that case it's allowed, but it is not a tag block in that case so the p tag that encloses `*but emd is allowed here*` will also enclose `*but not here*</section>`